packages/odyssey-react/src/components/Status/Status.tsx

```diff
@@ -63,10 +64,10 @@ let Status = forwardRef<HTMLDivElement, StatusProps>((props, ref) => {
   );
 
   return (
-    <div {...omitProps} className={styles.status} role="status" ref={ref}>
+    <Box {...omitProps} className={styles.status} role="status" ref={ref}>
       {labelElement}
       <span className={valueClass}>{descriptor}</span>
-    </div>
```

Comment:
sanity checking if you disable the global styles for typography and reset in storybook here does this work just fine?

Reply:
That is correct!